pkg/security/probe/probe_ebpf.go

```diff
@@ -822,11 +818,18 @@ func (p *EBPFProbe) handleEvent(CPU int, data []byte) {
 
 		pce := p.Resolvers.ProcessResolver.Resolve(event.CgroupWrite.Pid, event.CgroupWrite.Pid, 0, false, newEntryCb)
 		if pce != nil {
+			filesystem, _ := p.Resolvers.MountResolver.ResolveFilesystem(event.CgroupWrite.File.PathKey.MountID, event.CgroupWrite.File.Device, event.CgroupWrite.Pid, event.ContainerContext.ContainerID)
 			path, err := p.Resolvers.DentryResolver.Resolve(event.CgroupWrite.File.PathKey, true)
 			if err == nil && path != "" {
-				cgroupID := containerutils.CGroupID(filepath.Dir(string(path)))
+				if filesystem == "cgroup2" {
+					path = filepath.Dir(string(path))
+				}
+
+				cgroupID := containerutils.CGroupID(path)
```

Comment:
I thinks this is part from another PR, are you waiting it to be merged to rebase this one?

Reply:
Yes, the other PR should be merged soon